object/types.proto

```diff
@@ -36,8 +36,8 @@ message Header {
         Transform Transform                    = 4;
         // Tombstone header that set up in deleted objects
         Tombstone Tombstone                    = 5;
-        // Verify header that contains session public key and user's signature
-        session.VerificationHeader Verify      = 6;
+        // Token header that contains session token
```

Comment:
I would like to specify in comments, that token header contains session token, that was used during object creation.

Reply:
Yes, it would clarify the actual meaning. Done.